APCAppCore/APCAppCore/DataSubstrate/Model/APCUser+Bridge.m

```diff
@@ -81,6 +92,44 @@ - (void)signUpWithDataGroups:(NSArray<NSString *> *)dataGroups onCompletion:(voi
     }
 }
 
+- (void) signUpWithDataGroups:(NSArray<NSString *> *)dataGroups
```

Comment:
oh wait, never mind--I see you're using that for the "not testing" fall through case.